packages/flutter_tools/lib/src/android/android_studio_validator.dart

```diff
@@ -45,11 +50,11 @@ class AndroidStudioValidator extends DoctorValidator {
     final List<ValidationMessage> messages = <ValidationMessage>[];
     ValidationType type = ValidationType.missing;
 
-    final String? studioVersionText = _studio.version == null
-      ? null
-      : userMessages.androidStudioVersion(_studio.version.toString());
+    final String studioVersionText = _studio.version == null
+      ? 'version unknown'
+      : _userMessages.androidStudioVersion(_studio.version.toString());
     messages.add(ValidationMessage(
```

Comment:
Do we want to use the `_userMessages.androidStudioVersion()` with the string for version set to `unknown`? Seems like they both prefix `version` where needed

```suggestion
    final String studioVersionText = _studio.version == null
      ? _userMessages.androidStudioVersion('unknown')
      : _userMessages.androidStudioVersion(_studio.version.toString());
    messages.add(ValidationMessage(
```